.versionrc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "packageFiles": [
+    {
+      "filename": "version.json",
+      "type": "json"
+    }
+  ],
+  "bumpFiles": [
+    {
+      "filename": "version.json",
+      "type": "json"
+    }
+  ],
+  "commitAll": true,
+  "scripts": {
+    "postbump": "npx projen && git add ."
+  }
+}
```

Comment:
This file should make more sense once I practice releasing to npm on my private repo.

Reply:
To my understanding this is a file that allows you to configure Projen a bit more, specifically you can tell it where you want the version bumped, or scripts to run after bumping the version number.